### DISCUSSION TITLE
YOURLS一个ID捆绑多个域名

### DISCUSSION THREAD

**Discussion by yaobohan888** [YOURLS/YOURLS#3678] [Plugins]
YOURLS一个ID捆绑多个域名
// 原有的代码
$keyword = yourls_sanitize_string( $_REQUEST['id'] );

// 获取关联的所有长链接
$long_urls = get_long_urls_for_keyword( $keyword );

// 从关联的长链接中随机选择一个
$random_long_url = $long_urls[array_rand($long_urls)];

// 执行跳转
yourls_redirect($random_long_url, $keyword);
  是否可行，或者有没有类似的插件  大佬求助

**Comment by ozh** [comment]
English please.
